bindgen/features.rs

```diff
@@ -236,6 +239,9 @@ rust_feature_def!(
     Stable_1_47 {
         => larger_arrays;
     }
+    Stable_1_57 {
+        => const_panic;
```

Comment:
Could be lowered to 1.56 (gated by constification of [`core::mem::transmute`](https://doc.rust-lang.org/stable/core/mem/fn.transmute.html)) if we are okay with removing the `debug_asserts!` in the generated code.